src/de/dkfz/b080/co/aceseq/ACESeqWorkflow.java

```diff
@@ -103,15 +113,22 @@ public boolean execute(ExecutionContext context, BasicBamFile _bamControlMerged,
         return true;
     }
 
-    private boolean checkSvFile(ExecutionContext executionContext) {
-        // TODO getInitialBamFiles(executionContext)[0]
-        return true;
+    private boolean checkSvFileIfNecessary(ExecutionContext context) {
```

Comment:
Why not 
```
executable &= runWithSV(context) && checkSvFile(context)
```
The method name would get easier and the logic clearer, because it is directly clear, when the check is done (not just "if necessary").

Also the short-circuiting of the method by an early return -- one my my favorite anti-patterns -- would be unnecessary :-D

Reply:
I wanted to concentrate the check code at one place. I don't mind early return statements. But they should be directly in front of the metgid.. Or at the back. Not in between.

Reply:
If runWithSV == false, executable will be set to false, which is not correct. We'd need something like
executable &= !runWithSV(context) || (runWithSV(context) && (checkSvFile(context) || allowMissingSV(context))
Actually I prefer the method above then. What do you think?

Reply:
Right, my statement was incorrect. Indeed the correct statement is too complex. There would also be the option:
```groovy
if (runWithSV(context))
  executable &= checkSvFile(context)
```
That would at least express, that checkSvFile() is really checking something, without any other contextual information interfering in the check and would be more expressive than the "...IfNecessary" suffix. But I am not going into a fight over this :-P